flowkit_jwt_generator/flowkit_jwt_generator/jwt.py

```diff
@@ -0,0 +1,213 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import datetime
+import os
+import uuid
+from json import JSONEncoder
+from typing import Dict, List, Union, Callable
+from datetime import timedelta
+import click
+import jwt
+import pytest
+import requests
+
+
+permissions_types = {"run": True, "poll": True, "get_result": True}
+
+
+def generate_token(
```

Comment:
Should this function actually validate at least the basic structure of the claims dict?

Reply:
I was pondering that. My conclusion was that you want to be able to generate broken tokens for test purposes.

Reply:
Hmm, that's a good point. I feel that because our toolchains are fairly complicated already we should prioritise a good user experience and make "broken usage" for testing purposes possible but requiring an extra hoop to jump through. A useful way to do that could be to have a flag such as `allow_broken_claims=True` (probably with a better name for the argument). But again, I'm happy to leave that for a separate PR if you prefer.